<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyToolView</class>
 <widget class="QWidget" name="ProxyToolView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="historyTab">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="JournalView" name="journalView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="hexViewTab">
           <attribute name="title">
            <string>Hex View</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QHexEdit" name="hexView" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textViewTab">
           <attribute name="title">
            <string>Text View</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="textView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rulesTab">
      <attribute name="title">
       <string>Rules</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>Enabled</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Search</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Replace</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>qhexedit2/qhexedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JournalView</class>
   <extends>QTreeView</extends>
   <header>journalview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
